src/cljc/ataru/util.cljc

```diff
@@ -178,3 +178,7 @@
        (filter email-pred)
        (first)
        :value))
+
+(defn non-blank-val [m ks]
+  (when m
```

Comment:
Tässä vois ehkä käyttää `(some-> m (select-keys ks) (vals) #(filter not-blank? %) (first)`